### PR TITLE
Fix hosts file entry

### LINKED ISSUE #74: manage_hostsfile recipe not working
## :ghost: Brief Description
manage_hostsfile does not work as expected. The following error msg is encountered:
```
'hosts_file_entry' is not a valid Chef resource.

Please consult the documentation for a list of valid resources:

  https://docs.chef.io/resource_reference.html
```

## :pancakes: Cookbook version
1.0.1

## :woman_cook:  Chef-Infra Version
chef-client 16/17

## :tophat: Platform details
CentOS 7

## Steps To Reproduce

1. Create wrapper cookbook.
2. add `include_recipe 'dnsmasq'` to the default recipe
3. Set the following attributes:
```
default[:dnsmasq][:managed_hosts] = {
    '10.0.0.1' => 'a.random.hostname',
    '10.0.0.2' => 'evenmore.random.hostname'
  }
```

## :police_car: Expected behavior
Should create two entries in /etc/hosts that dnsmasq will resolv dns requests for.

### PR DESCRIPTION
**Pull request by OperationalDev**
# Description

Describe what this change achieves

## Issues Resolved

#74

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: metadata.rb [hunk 1]
```diff
@@ -9,8 +9,11 @@
 issues_url        'https://github.com/sous-chefs/dnsmasq/issues'
 version           '1.1.0'
 
+depends 'hostsfile'
```
Comment (damacus) [inline metadata.rb:12]:
```suggestion
depends 'hostsfile'
```

**Comment by damacus** [comment]
Looks like this one is a bit of a rabbit hole.we should fix the hostile cookbook first

**Comment by OperationalDev** [comment]
Yeah that was my initial thought. However, it hasn't had a MR approved/merged since 2017 (https://github.com/customink-webops/hostsfile). It may need to be adopted.

**Comment by damacus** [comment]
Ah, actually we can remove the dependency completely probably. 


https://docs.chef.io/resources/hostname/

**Pull request closed without merging**